packages/lesswrong/components/sunshineDashboard/SunshineCuratedSuggestionsList.tsx

```diff
@@ -4,15 +4,20 @@ import { useMulti } from '../../lib/crud/withMulti';
 import { Posts } from '../../lib/collections/posts';
 
 const SunshineCuratedSuggestionsList = ({ terms }) => {
-  const { results, loading } = useMulti({
+  const { results, loading, count, totalCount, loadMore } = useMulti({
     terms,
     collection: Posts,
-    fragmentName: 'PostsList'
+    fragmentName: 'PostsList',
+    enableTotal: true,
+    itemsPerPage: 60
   });
   
   if (loading) return <Components.Loading/>;
   
-  const { SunshineListTitle, SunshineCuratedSuggestionsItem, LastCuratedDate } = Components
+  const { SunshineListTitle, SunshineCuratedSuggestionsItem, LastCuratedDate, LoadMore } = Components
+  
+  const showLoading = totalCount && count && totalCount > count
```

Comment:
`useMulti` returns a `showLoadMore` field, which you should use rather than computing this yourself.